### DISCUSSION TITLE
connect erp2frepple

### DISCUSSION THREAD

**Discussion by atistabiur** [frePPLe/frepple#538] [Q&A]
if i understand correctly the wait that erp2freppe is running, when i press the button ,freeple is going to my database, run a SQL query  to extract the required data and places in a folder.

When i run erp2frepple i get this error:

(InterfaceError("Windows COM Error: Dispatch('ADODB.Connection') failed."), 'Error opening connection to "Provider=SQLNCLI11;Server=localhost;Database=acutec;User Id=acutec;Password=acutec;"')	

i think  is asking for my database connection authentication
- When do i configure these setting ?
-  where is the file be dropped ? at : /var/log/freeple ?

i am asking these questions because i am trying to integrate freeple community version in a ERP software (PHC) and see out it behaves and if this is the right tool for me.

thanks for the support

**Comment by jdetaeye** [comment]
>> When do i configure these setting ?

A search on github shows you the location: https://github.com/frePPLe/frepple/search?q=acutec
It's currently hardcoded.
A cleaner approach would be to add a setting for it in the file "/etc/frepple/djangosettings.py" and pick up that setting from the connector.

>> where is the file be dropped ? at : /var/log/freeple ?

The extracted files are located at "/var/log/frepple/data/<scenario>".
That's also where the "import data files" task (see https://frepple.org/docs/6.23/command-reference.html#importfromfolder)  picks them up from.

**Comment by hichamlahlou** [comment]
erp2frepple is supposed to connect to your ERP database (not your frepple
database), execute SQL queries there and generate csv file for each frepple
entity (items, locations, customers...).
Obviously, the sql for each entity (in the extractItem, extractLocation,
extractCustomer....functions) will have to be adapted to your ERP.
The connection error you are getting is because the code looks for a
Microsoft SQL database on your local machine, which I assume, you don't
have. So you need to adapt that string to reflect the type of database your
ERP is using and provide credentials.
The output folder is the FILEUPLOADFOLDER variable configured in your
djangosettings.py file.

Hicham
